src/index.js

```diff
@@ -0,0 +1,145 @@
+let temperature = 48;
+
+const tempValue = document.getElementById('tempValue');
+const increaseTempControl = document.getElementById('increaseTempControl');
+const decreaseTempControl = document.getElementById('decreaseTempControl');
+const landScape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset');
+const headerCityName = document.getElementById('headerCityName');
+const getRealTimeTempButton = document.getElementById('getRealTimeTempButton');
+
+//Update the sky options when we choose different sky and change the color background in garden
+const skyOptions = {
+    'sunny': {
+        'text': "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+        'backgroundColor': "#DFFFFF",
+    },
+    'cloudy': {
+        'text': "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+        'backgroundColor': "#D3D3D3",
+    },
+    'rainy': {
+        'text': "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+        'backgroundColor': "#B0C3DF",
+    },
+    'snowy': {
+        'text': "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+        'backgroundColor': "#ADD9E6",
+    },
+};
```

Comment:
Nice way of storing these! You could even put them in another file and just import them to better declutter your code!